### ISSUE TITLE
Crash when banner widget is dismissed before banner was loaded

### ISSUE THREAD

**Issue by Koallider** [Koallider/flutter_yandex_mobile_ads#13]
<img width="523" alt="report" src="https://user-images.githubusercontent.com/2699619/208167225-154211da-e52c-4135-9afc-c405f4d97d10.png">

**Comment by Koallider** [comment]
``` 
Unhandled Exception: MissingPluginException(No implementation found for method reportSize on channel me.koallider.flutter_yandex_mobile_ads/banner38)
E/flutter (13193): #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:294:7)
E/flutter (13193): <asynchronous suspension>```

**Comment by Koallider** [comment]
fixed in #14
